doc/classes/KinematicBody2D.xml

```diff
@@ -98,12 +111,13 @@
 			<argument index="1" name="rel_vec" type="Vector2">
 			</argument>
 			<description>
-				Returns true if there would be a collision if the body moved from the given point in the given direction.
+				Returns true if there would be a collision if the body started at the given position, with the given rotation and scale, and then moved along the given vector.
```

Comment:
Needs to change. Suggestion:
`Virtually moves the node along the vector. If it doesn't encounter any obstacle, returns [code]true[/code]. If it collides with another body, returns [code]false[/code]. Use [method move_and_slide] to move the node directly.`